### PR TITLE
fix(&nbsp): a bug of header contains the &nbsp fixed

### LINKED ISSUE #257: [ Bug ] `&nbsp;` in tab title when note header contains spaces
Image

### PR DESCRIPTION
**Pull request by dependentmadani**
## Problem:
When the first input in the note contains spaces, the header of the note show `&nbsp;`. 
resolves #257 

## Cause:
The noteContent get stored as HTML entity. For this reason, the title shows the `&nbsp;`, whenever there is occurrence of space in the first block of the note.

## Solution:
- Target the return value of function in `useNote` service and replace all occurrences of `&nbsp;` with space.
- The change was made also on the `NoteList` where the bug occurs too.

### CODE REVIEW

**Comment by neSpecc** [comment]
Please, attach an issue correctly

**Comment by dependentmadani** [comment]
The attachment has been updated.

File: src/application/services/useNote.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     if (!firstNoteBlock || !Boolean(firstNoteBlock.data.text)) {
```
Comment (neSpecc) [inline src/application/services/useNote.ts:147]: it would be better if we will reuse the `getTitle()` util here

Comment (dependentmadani) [reply src/application/services/useNote.ts:147]: I think there will be an issue of using I18n at the top of a `setup` function.

Comment (neSpecc) [reply src/application/services/useNote.ts:147]: you can pass a `defaultTitle` as an optional argument, for example

**Review by neSpecc — APPROVED** [review]

File: src/application/services/useNote.ts [hunk 2]
```diff
@@ -128,7 +129,7 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
 
   const route = useRoute();
 
-  const limitCharsForNoteTitle = 50;
+  // const limitCharsForNoteTitle = 50;
```
Comment (e11sy) [inline src/application/services/useNote.ts:132]: remove unused comments

**Review by neSpecc — APPROVED** [review]

**Review by e11sy — APPROVED** [review]

**Pull request merged**